### PR TITLE
expansion of camel on springboot for ocp 4.18 fips/non-fips

### PR DESCRIPTION
**Pull request by vi-patel**
update cron
  add FIREWATCH_DEFAULT_JIRA_ASSIGNEE
  create .config.prowgen file for slack reporting of results being auto-created with make updates

### CODE REVIEW

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.18-lp-interop-csb-interop-aws-latest

**Comment by vi-patel** [comment]
test dependency issue, need PQE investigation

**Comment by dfrazzette** [comment]
Non-FIPS ([1843681881220976640](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/57608/rehearse-57608-periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.18-lp-interop-csb-interop-aws-latest/1843681881220976640)) and FIPS ([1843701816508813312](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/57608/rehearse-57608-periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.18-lp-interop-csb-interop-aws-latest-fips/1843701816508813312)) rehearsal runs failed due to dependency on missing `software.tnb:system-x-flink:jar`  file. Automation succeeded, so merging PR to include in OCP 4.18 test suite. Will investigate any duplicate failures during monthly FIPS/weekly non-FIPS run until operator and changes are available.

File: ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml [hunk 1]
```diff
@@ -456,3 +456,187 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml:460]:
```suggestion
  cluster: build10
```

File: ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml [hunk 2]
```diff
@@ -456,3 +456,187 @@ periodics:
...
+      secret:
+        secretName: manifest-tool-local-pusher
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml:552]:
```suggestion
  cluster: build10
```

**Pull request merged**